source/threads.tex

```diff
@@ -4970,10 +4976,8 @@
 \begin{itemdescr}
 \pnum
 \expects
-\tcode{order} is neither
-\tcode{memory_order::consume},
-\tcode{memory_order::acquire}, nor
-\tcode{memory_order::acq_rel}.
+The \tcode{order} argument is either \tcode{memory_order::relaxed},
```

Comment:
I believe "either" in English preferentially refers to a choice of two things.  We have three things here.
I'd suggest to simply drop "either"; I don't think it adds anything.

Reply:
OK.